school/classroom.rb

```diff
@@ -0,0 +1,13 @@
+class ClassRoom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @student.push(student)
+  end
```

Comment:
- [ ] What you declared is a `@students` array not `@student`. Kindly fix this error.

- [ ] Apart from adding the student to the array you still need to set the classroom of the student to this classroom to complete the association.

  ```
  def add_student(student)
    @students.push(student)
    student.classroom = self
  end
  ```